frontend/src/components/Vote/base.vue

```diff
@@ -1,8 +1,8 @@
 <template>
     <div class="meta-fields">
-        <p v-on:click="upvote" class="material-icons vote">keyboard_arrow_up</p>
+        <p v-on:click="upvote" :class="[userLiked === 1 ? 'voted' : '', 'material-icons vote']">keyboard_arrow_up</p>
         <p class="likes">{{ likes }}</p>
-        <p v-on:click="downvote" class="material-icons vote">keyboard_arrow_down</p>
+        <p v-on:click="downvote" :class="[userLiked === -1 ? 'voted' : '','material-icons vote']">keyboard_arrow_down</p>
```

Comment:
It took me a little while to actually notice the colour change of the arrows... Can we make a more noticeable change?

Reply:
I think it's cos `--theme` is a similar tone to the regular grey color, maybe try --theme-light or something in between?
Also, maybe the down arrow should be red/orange after voting